lib/logstash/outputs/sqs/patch.rb

```diff
@@ -0,0 +1,18 @@
+# This patch was stolen from logstash-plugins/logstash-output-s3#102.
```

Comment:
@jrgns Lets make a PR for adding this patch directly to mixin, but from what I've heard this patch is only needed because the AWS ruby sdk does funky thing with autoload and on some ruby implementation or concurrency situation it will/can fail. 

Last I've heard amazon are completely removing the autoload logic for the greather good.